### PR TITLE
[Homepage] Optical Design Section

### LINKED ISSUE #47: [Homepage] Optical Design Section
Style Optical Design Section with Cloumns ( icons ) with section metadata to add orange color scheme for icons and links

Screenshot 2023-04-11 at 12 35 57 PM

### PR DESCRIPTION
**Pull request by agangam**
Fix #47 

Test URLs:
- Before: https://main--zemax--hlxsites.hlx.page/
- After: https://issue-47-optical--zemax--hlxsites.hlx.page/

After the changes :
image

### CODE REVIEW

File: scripts/lib-franklin.js [hunk 1]
```diff
@@ -534,7 +535,8 @@ export function decorateTemplateAndTheme() {
 export function decorateButtons(element) {
   element.querySelectorAll('a').forEach((a) => {
     a.title = a.title || a.textContent;
-    if (a.href !== a.textContent) {
+    // skip buttons for 'Learn More'
+    if (a.textContent.startsWith('Learn More') === false && a.href !== a.textContent) {
```
Comment (agangam) [inline scripts/lib-franklin.js:539]: This seemed like a reasonable approach to skipping buttons. All the 'Learn More' usages are not buttons

File: styles/styles.css [hunk 2]
```diff
@@ -439,15 +465,112 @@ main .divider{
...
+}
+
+.optical-design .columns > div > div > p:first-child {
+  padding-top: 1em;
+  padding-left: 0.5em
+}
+
+.optical-design .columns > div > div > p > span {
+  width: 45px;
+  height: 60px;
```
Comment (agangam) [inline styles/styles.css:532]: Each of the icons has a different size on it. I used the size 45x60 for all three of them. The other sizes are 32x63 and 47x63

File: scripts/lib-franklin.js (hunk 1 above)
Comment (kunwarsaluja) [reply scripts/lib-franklin.js:539]: @agangam would it make sense to override the button styling as section style eg.  ‘no-button-decoration’ or something that is intuitive ? coz there are other areas where we have other color schemes for links.

File: styles/styles.css (hunk 2 above)
Comment (agangam) [reply styles/styles.css:538]: @kmurugulla Changed the icon sizes to match zemax.com
ed407de6d2dee557dc8c931af9e08967e66f69d5

File: styles/styles.css [hunk 3]
```diff
@@ -104,14 +104,18 @@ body {
   font-size: var(--body-font-size-m);
   margin: 0;
   font-family: var(--body-font-family);
-  line-height: var(--line-height-160);
+  line-height: var(--line-height-16);
```
Comment (agangam) [inline styles/styles.css:107]: This seems like a typo, so corrected it

File: scripts/lib-franklin.js (hunk 1 above)
Comment (kunwarsaluja) [reply scripts/lib-franklin.js]: Yes that is correct.

Something seems off on the layout as this section should be --section-width and currently is taking over the view-width

Screenshot 2023-04-19 at 2 39 30 PM
Screenshot 2023-04-19 at 2 39 44 PM

Comment (agangam) [reply scripts/lib-franklin.js]: Changed width to 1000px as the heading was going to over to next line with --section-width.

After changes : 
image

**Review by kunwarsaluja — APPROVED** [review]

**Pull request merged**